README.md

```diff
@@ -51,29 +51,33 @@ At Babylon, we firmly believe that **transparency** is a core value that should
-| Professional Services         | Chitra             |    1/3      | [#telus](https://babylonhealth.slack.com/archives/CAJ7YQZ5Z) |
-| Enrolment and Integrity       | Diego                            |    1/1       | [#ei-squad](https://babylonhealth.slack.com/archives/CGR4D5NKX) |
-| Core Experience               | Sergey, Yuri                     |    2/2       | [#core-experience](https://babylonhealth.slack.com/archives/CCSE8JLK0) |
-| Health Management             | David, João                      |    2/2       | [#health-mgmt-public](https://babylonhealth.slack.com/archives/CCNHJUXLH) |
-| Monitor                       | Anders, Joshua, Nicola           |    3/3       | [#patient-metrics-team](https://babylonhealth.slack.com/archives/CE37S5W9Z) |
-| Tests and Kits                | Michal                           |    1/1       | [#test-and-kits](https://babylonhealth.slack.com/messages/CGCDJA8CD) |
-| Triage UI                     | Michael, Danilo    |    2/2       | [#squad_triage_ui](https://babylonhealth.slack.com/archives/CE6H6SLRX) |
-| Women's Health                | Anil                             |    1/1       | [#womenshealthteam](https://babylonhealth.slack.com/archives/CFX73S4P5) |
+| Squad Name                    | Members                              | Availability | Public slack channel |
+|-------------------------------|--------------------------------------| ------------ | -------------------- |
+| SDK                           | Viorel, Simon                        |    2/3       | [#sdk_squad](https://babylonhealth.slack.com/archives/CC5JNDGJJ) |
+| Consultation                  | Ilya, Adrian                         |    2/2       | |
+| Booking                       | Witold                               |    1/1       | |
+| Prescriptions                 | Adam, Konrad                         |    2/2       | [#prescription-squad](https://babylonhealth.slack.com/archives/C88TCM9JB) |
+| Healthcheck                   | Ben, Catarina, Julien                |    3/3       | [#healthcheck_tribe](https://babylonhealth.slack.com/archives/C7995CX3R) |
+| Native/Core                   | Giorgos, Jason, Martin, Inami, Emese |    5/6       | [#nativeapps_public](https://babylonhealth.slack.com/archives/CE5P8LRNH) |
+| Professional Services         | Chitra                               |    1/3       | [#telus](https://babylonhealth.slack.com/archives/CAJ7YQZ5Z) |
+| Enrolment and Integrity       | Diego                                |    1/1       | [#ei-squad](https://babylonhealth.slack.com/archives/CGR4D5NKX) |
+| Core Experience               | Sergey, Yuri                         |    2/2       | [#core-experience](https://babylonhealth.slack.com/archives/CCSE8JLK0) |
+| Health Management             | David, João                          |    2/2       | [#health-mgmt-public](https://babylonhealth.slack.com/archives/CCNHJUXLH) |
```

Comment:
@dmcrodrigues you are no longer in Health Management, are you?

Reply:
I'm not, I'll open a follow up to change this and include @AnnKatF.